daffodil-core/src/test/scala-new/edu/illinois/ncsa/daffodil/dsom/TestIsScannable.scala

```diff
@@ -0,0 +1,143 @@
+/* Copyright (c) 2012-2015 Tresys Technology, LLC. All rights reserved.
```

Comment:
Year is wrong here, and I think this should probably be the Apache license now. All new code should be license under Apache. The new header is:
```
/*
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
```

Reply:
Should we be adding this banner to files we edit, that are just being modified, but not new?

Reply:
I believe only new files should have the new banner.